### ISSUE TITLE
[Bug]: Lilypad of Fertility stops affecting crops over time

### ISSUE THREAD

**Issue by rowe-boat** [AllTheMods/ATM-10#1627]
### Possible Fixes

Yes

### Modpack Version

2.6

### What happened?

I have a few mystical agriculture crop farms that perform great, until without warning, I realize that my crops are back growing at a snail's pace-- as if the lilypads aren't there whatsoever. I'm playing with two other folks on this server and they have not experienced this issue. I've moved my farms, completely torn down and rebuilt the farms-- the only fix I've found is completely breaking all Lilypads and placing them back on the water's surface.  I'll attach a snippet showing that the height is fine from pad to crop.
![image](https://github.com/user-attachments/assets/16b24e92-bfac-4380-be21-7d35157dae70)

**Comment by number65259** [comment]
I experienced this right after a recent server restart. It seems almost like all the lilypads are contributing to the same growth stage across all crops. When this happens to me, throughput tanks but is higher than with 0 lilypads, and all crops start to mature at the same time.

**Comment by rowe-boat** [comment]
Agreed, they all start growing (incredibly slow) at the same pace it seems.

**Comment by SharkBurn** [comment]
Same issue here. When first placed down the lily pads will affect crop growth like normal, but after a bit they will only affect the crops at the same time, instantly growing all of them. Still beneficial, but bottlenecks the growth speed of all the crops to the time between effects.

**Comment by rowe-boat** [comment]
I crafted all new lily pads to attempt to rectify any issues. For transparency, when I first had my crops planted, I decided to move them to a different area and I attempted using the cut paste tool, this pulled almost everything from my original garden. 
I wondered if this could be my issue, but after crafting all new lily pads, the issue still persists.

It should also be noted that we upgraded to version 2.16 when it was the current most up-to-date version. I'm at a loss trying to figure out why this issue exists for myself, but not my server mates. Could it be somehow biome based?

**Comment by NediaMedia** [comment]
Still having this issue with the lastest version of the pack? Not sure if there is any fix for this.

**Comment by RealCaptainObvious** [comment]
Still having this issue in version "All the Mods 10-2.44" (most recent version when posting). no solution still?

**Comment by Plaperr** [comment]
I had the same issue and i think i found a solution. At first i tried to replace them regularly with an RFTools Builder, but placing them all at the same time didnt help, as the crops immediatly had the problem of all growing simultaniously.

I realised the problem is probably, that if the lilypads are all loaded at the same time, they sync their "growth tick" so they all boost the plant at once, creating this problem. If the chunk is unloaded and loaded again, the same thing seems to happen.

The FTB Teams chunkloading doesnt seem to work in Version 2.44, **so i placed Chunky mining turtles next to the farm and that fixed the problem**. Not sure if my reasoning is correct, but it resolved the issue, and thats enough for me ^^
